src/librustc_trait_selection/traits/error_reporting/mod.rs

```diff
@@ -559,6 +559,13 @@ impl<'a, 'tcx> InferCtxtExt<'tcx> for InferCtxt<'a, 'tcx> {
                             obligation
                         )
                     }
+
+                    ty::Predicate::WellFormedConst(ty) => {
+                        // const WF predicates cannot make errors themselves.
+                        //
+                        // So this should be unreachable
+                        span_bug!(span, "const WF predicate not satisfied for {:?}", ty);
+                    }
                 }
             }
 
```

Comment:
Missing unhandled `WellFormedConst` in this file: https://github.com/rust-lang/rust/blob/f4c675c476c18b1a11041193f2f59d695b126bc8/src/librustc_trait_selection/traits/error_reporting/mod.rs#L1446-L1453